src/lib/classes/access-control-list.test.js

```diff
@@ -0,0 +1,48 @@
+import AccessControlList from './access-control-list';
+import ldflex from '../../test/__mocks__/@solid/query-ldflex';
+
+const webIdExample = 'https://example.org/#me';
+const fileExample = 'https://example.org/public/test.ttl';
+
+describe('Access Control List test', () => {
+  beforeAll(() => {
+    jest.restoreAllMocks();
+  });
+
+  const PERMISSIONS = {
```

Comment:
Can this list be pulled from somewhere else?

Reply:
done